projects/github.com/helmfile/vals/package.yml

```diff
@@ -0,0 +1,28 @@
+distributable:
+  url: https://github.com/helmfile/vals/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: helmfile/vals
+
+provides:
+  - bin/vals
+
+build:
+  dependencies:
+    go.dev: '*'
+  script:
+   - go mod download
+   - mkdir -p "{{ prefix }}"/bin
+   - go build -trimpath -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/vals ./cmd/vals
+  env:
+    LDFLAGS:
+      - -w
+      - -s
+      - -X main.version=v{{version}}
```

Comment:
probably need to add
```yaml
    linux:
      ARGS:
        - -buildmode=pie
```

if using cgo, to prevent segfaults.

Reply:
Any recommendation? Should we prefer building with cgo or without? In their makefile, vals have cgo enabled.

Reply:
Mmh, I was looking to the Makefile, however, goreleaser builds the production releases

https://github.com/helmfile/vals/blob/57ae35b137ae8b95b979b262df81433eb349b84a/.goreleaser.yml#L6

CGO is disabled

Reply:
there are some [interesting arguments against it](https://dave.cheney.net/tag/cgo), but if it works, it works. ;)